scripts/ansible

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Note: don't forget to login to registry.gitlab.com: `docker login registry.gitlab.com`
+
+ANSIBLE_NAME=${ANSIBLE_NAME:-ansible}
+ANSIBLE_IMG=registry.gitlab.com/mshahbaz-devops/ansible-template/ansible:2.12.1
+ANSIBLE_MOUNT_DIR=${ANSIBLE_MOUNT_DIR:-${PWD}}
+
+docker run -it --rm \
+  --name ${ANSIBLE_NAME} \
+  --network host \
+  -v "${ANSIBLE_MOUNT_DIR}":/workdir -w /workdir \
+  ${ANSIBLE_IMG} "$@"
```

Comment:
Why do we need all of this?

Reply:
this is not needed, from gnbsim/oai blueprints they kept these targets. So i just kept in the same way. But i agree this is not needed and should be removed. I just updated.